cameo/strain_design/heuristic/evolutionary/optimization.py

```diff
@@ -43,7 +45,9 @@
 __all__ = ['ReactionKnockoutOptimization', 'GeneKnockoutOptimization']
```

Comment:
add `CofactorSwapOptimization` to `__all__`. Also change `from cameo.strain_design.heuristic.evolutionary.optimization import ReactionKnockoutOptimization, \
    GeneKnockoutOptimization` in evolutionary/__init__.py to `from cameo.strain_design.heuristic.evolutionary.optimization import *`